src/System.Private.CoreLib/src/System/Runtime/InteropServices/CallbackAlreadyRegisteredException.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace System.Runtime.InteropServices
+{
+    public class CallbackAlreadyRegisteredException : System.ArgumentException
```

Comment:
why derived from System.ArgumentException?

Reply:
It should definitely derive from System.Exception